obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  mp_starcat_time: 2007:302:21:17:28.912
+  status: 1
+- agasc_id:
+  - 34997808
+  comments: Venus observation
+  mp_starcat_time: 2007:302:23:45:40.189
+  status: 1
+- agasc_id:
+  - 35398128
+  comments: Venus observation
+  mp_starcat_time: 2007:303:01:44:10.189
+  status: 1
+- agasc_id:
+  - 35001864
+  comments: Venus observation
+  mp_starcat_time: 2007:303:03:42:40.189
+  status: 1
+- agasc_id:
+  - 35004328
+  comments: Venus observation
+  mp_starcat_time: 2007:303:03:42:40.189
+  status: 1
+- agasc_id:
+  - 35004456
+  comments: Venus observation
+  mp_starcat_time: 2007:303:05:41:10.189
+  status: 1
+- agasc_id:
+  - 38535536
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2008:057:22:07:34.020
+  status: 1
+- agasc_id:
+  - 185871616
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2008:073:16:53:06.691
+  status: 1
+- agasc_id:
+  - 974791856
+  comments: Faint star. Poorly tracked. Set mag to MAXMAG
+  mp_starcat_time: 2008:183:05:33:47.836
+  status: 1
+- agasc_id:
+  - 978846152
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978848616
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978849272
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978851616
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978853160
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978986608
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 1043726440
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 1043726952
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 639635448
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:023:22:13:56.794
+  status: 1
+- agasc_id:
+  - 639635448
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:024:04:12:45.955
+  status: 1
+- agasc_id:
+  - 395055600
+  comments: UNKNOWN: Never acquired. Unknown issue with observation. Presume star is OK.
+  mp_starcat_time: 2009:040:05:37:54.381
+  status: 1
+- agasc_id:
+  - 720504096
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:049:20:22:15.791
+  status: 1
+- agasc_id:
+  - 185871616
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:073:15:34:21.422
+  status: 1
+- agasc_id:
+  - 616699752
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:290:21:44:31.992
+  status: 1
+- agasc_id:
+  - 373564272
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:335:11:29:27.547
+  status: 1
+- agasc_id:
+  - 161357600
+  comments: Almost never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:345:18:09:20.636
```

Comment:
This seems fine, but I also note there are only 133 samples in this observation.  I think that is enough to determine that the "never got the star" is useful information but I'm not sure of the minimum number of samples required.

Reply:
I am not sure exactly what you are proposing here.

Reply:
I think I was proposing no change; just asking if we had a minimum dwell duration to be included in the table(s).